bears/general/InvalidLinkBear.py

```diff
@@ -8,15 +8,16 @@
 
 
 class InvalidLinkBear(LocalBear):
+    check_connection_url = "http://www.google.com"
```

Comment:
In wondering here ...
Maybe we want a better check where we test for 8.8.8.8 and then check for google.com?
This way we can test if the dns server is not there or the internet connection itself is not there ?

-----Original Message-----
From: "Sanket Dasgupta" <notifications@github.com>
Sent: ‎16-‎03-‎2016 08:41
To: "coala-analyzer/coala-bears" <coala-bears@noreply.github.com>
Subject: Re: [coala-bears] InvalidLinkBear: Improve request handling (#242)

In bears/general/InvalidLinkBear.py:
> @@ -8,15 +8,16 @@
>  
>  
>  class InvalidLinkBear(LocalBear):
> +    check_connection_url = "http://www.google.com"
Yeah its actually 216.58.218.174. I'll add this and add a comment.
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub